cmd/api-converter/main.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"asyncapi-converter/pkg/asyncapi"
+	"asyncapi-converter/pkg/converter/v2rc1"
+	"net/http"
+	"net/url"
+
+	"github.com/docopt/docopt-go"
+	"github.com/pkg/errors"
+
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+const version = "asyncapi-converter 0.1.0rc1"
+
+var (
+	errFileDoesNotExist = errors.New("file does not exist")
+	errInvalidArgument  = errors.New("invalid argument")
+)
+
+type helper struct {
```

Comment:
Maybe extract helper and related functions to this struct to separate package?

Reply:
Maybe `commandSerializer` would be better name?

Reply:
the helper make is usable only for main package; we can leave the same package (main) but move it to the different file